src/views/SettingsView.vue

```diff
@@ -133,6 +133,7 @@ export default {
         try {
           const parsedJson = JSON.parse(this.importJsonCode)
           await this.setWeapons(parsedJson)
+          await this.setSchematics(parsedJson)
```

Comment:
This probably won't work since we're only exporting the weapons (and camouflages) progress into the JSON-code. I'll create an issue for it and see if I find the time to include calling cards and schematics as well (when that gets merged) 😁